docs/tutorial/progress-bar.md

```diff
@@ -24,7 +32,7 @@ currently at 63% towards completion, you would call it as
 
 Setting the parameter to negative values (e.g. `-1`) will remove the progress
 bar, whereas setting it to values greater than `1` (e.g. `2`) will switch the
-progress bar to intermediate mode (100% complete).
+progress bar to indeterminate mode (100% complete).
```

Comment:
It seems like indeterminate mode is a Windows-only feature, and represents a progress bar that's active but doesn't show an actual percentage. We should make that clear in this line (Windows-only and doesn't represent 100% complete)

Ref: https://docs.microsoft.com/en-us/dotnet/api/system.windows.shell.taskbaritemprogressstate?view=netcore-3.1#System_Windows_Shell_TaskbarItemProgressState_Indeterminate